great_expectations/cli/example.py

```diff
@@ -19,6 +19,68 @@ def example() -> None:
     pass
 
 
+@example.command(name="snowflake")
+@click.option(
+    "--stop",
+    is_flag=True,
+    help="Stop example and clean up. Default false.",
+    default=False,
+)
+@click.option(
+    "--url",
+    is_flag=True,
+    help="Print url for jupyter notebook.",
+    default=False,
+)
+def example_snowflake(stop: bool, url: bool) -> None:
+    """Start a snowflake database example."""
+    repo_root = pathlib.Path(__file__).parents[2]
+    example_directory = repo_root / "examples" / "reference_environments" / "snowflake"
+    assert example_directory.is_dir(), "Example directory not found"
+    if stop:
+        cli_message("<green>Stopping example containers...</green>")
+        stop_commands = ["docker", "compose", "down"]
+        subprocess.run(stop_commands, cwd=example_directory)
+        cli_message("<green>Done stopping containers.</green>")
+    elif url:
+        container_name = "gx_snowflake_example_jupyter"
+        url_commands = [
+            "docker",
+            "exec",
+            container_name,
+            "jupyter",
+            "server",
+            "list",
+            "--json",
+        ]
+        url_json = subprocess.run(
+            url_commands,
+            cwd=example_directory,
+            capture_output=True,
+        ).stdout
+        raw_json = json.loads(url_json)
+        notebook_url = (
+            f"http://127.0.0.1:{raw_json['port']}/lab?token={raw_json['token']}"
+        )
+        cli_message(f"<green>Url for jupyter notebook:</green> {notebook_url}")
```

Comment:
Not a blocker in this PR, but let's be mindful as we add more of these reference environments we can probably refactor some of this duplicated logic.